tests/python/unittest/test_numpy_op.py

```diff
@@ -1650,6 +1650,7 @@ def hybrid_forward(self, F, a, b, *args, **kwargs):
         'power': (1.0, 2.0, [lambda y, x1, x2: _np.power(x1, x2 - 1.0) * x2],
                              [lambda y, x1, x2: _np.power(x1, x2) * _np.log(x1)]),
         'lcm': (-100, 100, [None], None, [[_np.int32]]),
+        'bitwise_xor': (-100, 100, [None], None, [[_np.int32]]),
```

Comment:
Please add `bitwise_xor` to `_NUMPY_ARRAY_UFUNC_LIST` and test cases from the official numpy codebase to `test_numpy_interoperability.py`.